docs/spfx/build-for-teams-overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Build for Microsoft Teams using SharePoint Framework
+description: By building Microsoft Teams applications using SharePoint Framework, you can save costs on hosting infrastructure and simplify the deployment and operation process.
+ms.date: 06/15/2020
+ms.prod: sharepoint
+localization_priority: Normal
+---
+
+# Build for Microsoft Teams using SharePoint Framework
+
+You can extend Microsoft Teams with additional functionality by integrating your applications. This allows you expose functionality in the context of your users’ work helping them be more productive. By [building Microsoft Teams applications using SharePoint Framework](integrate-with-teams-introduction.md), you can save costs on hosting infrastructure and simplify the deployment and operation process.
+
+> [!TIP]
+> To see how to use the different concepts described in this article, see the sample [Leads application](https://github.com/pnp/sp-dev-solutions/tree/master/solutions/LeadsLOBSolution) on GitHub.
+
+## Why build for Microsoft Teams using SharePoint Framework
+
+There are two main benefits for using the SharePoint Framework to build for Microsoft Teams: simplified hosting and ease of integration with APIs secured with Azure Active Directory.
+
+### Automatic solution hosting
+
+When you build solutions for Microsoft Teams, you need to host them somewhere. Typically, it means creating and monitoring an Azure App Service. It also means that you now have extra infrastructure for which you have to pay and continuously monitor.
+
+When you build Teams tabs and personal apps using SharePoint Framework, your solutions are automatically hosted for you in SharePoint. You don’t need any additional resources in Microsoft Azure. This significantly simplifies the deployment process and operations and lowers the overall cost of your solution.
+Additionally, if you have Office 365 CDN enabled, assets for your Teams tabs and personal apps will be served from the CDN offering your users significantly better experience without any extra cost or configuration.
```

Comment:
Missing line break between paragaphs

Reply:
This was intentional. It was meant as a line break rather than a separate paragraph. If that's unclear, I'll make it into a separate paragraph.